dast/__main__.py

```diff
@@ -29,15 +29,16 @@ def main(then_path: str, now_path: str, verbose: bool = True):
         return None, None
     with _open_utf(then_path) as f_then:
         then_ast = ast.parse(f_then.read())
-        strip(then_ast)
         prettify(then_ast)
 
     with _open_utf(now_path) as f_now:
         now_ast = ast.parse(f_now.read())
-        strip(now_ast)
         prettify(now_ast)
 
-    diff = DeepDiff(then_ast, now_ast)
+    # This also includes end_lineno and end_col_offset
```

Comment:
How does it include them?

Reply:
Because `path.endswith(prop)`  with `prop=='col_offset'` also fires when `path.endswith('end_col_offset')`

Reply:
:facepalm: yep